envoy/Makefile

```diff
@@ -128,7 +183,7 @@ endif
 endif
 
 force-non-root:
-ifeq ($(USER),root)
+ifneq ($(USER),root)
```

Comment:
This flips the test while it should not.

Reply:
Doh, left over from my manual testing.
The CI test failed because of that.
Sorry, will fix this.